utils/misc.py

```diff
@@ -151,3 +151,37 @@ def download_ampere_imagenet():
 
     os.environ["IMAGENET_IMG_PATH"] = str(dataset)
     os.environ["IMAGENET_LABELS_PATH"] = str(labels)
+
+
+def download_widedeep_processed_data(batch_size):
```

Comment:
if this is a util dedicated to a particular model I think it should be under that model's directory